tests/foreman/virtwho/utils.py

```diff
@@ -180,43 +236,47 @@ def _get_hypervisor_mapping(logs):
         )
 
 
-def deploy_validation(debug=True):
+def deploy_validation():
     """Checkout the deploy result
-    :param bool debug: if VIRTWHO_DEBUG=0, this option should be False,
-        because there is no json data logged in rhsm.log.
     :raises: VirtWhoError: If failed to start virt-who servcie.
+    :ruturn: hypervisor_name and guest_name
     """
     status = get_virtwho_status()
     _, logs = runcmd('cat /var/log/rhsm/rhsm.log')
     error = len(re.findall(r'\[.*ERROR.*\]', logs))
     if status != 'running' or error != 0:
         raise VirtWhoError("Failed to start virt-who service")
-    if debug:
-        hypervisor_name, guest_name = _get_hypervisor_mapping(logs)
-        # Delete the hypervisor entry and always make sure it's new.
-        hosts = Host.list({'search': hypervisor_name})
-        for index, host in enumerate(hosts):
-            Host.delete({'id': host['id']})
-        runcmd("systemctl restart virt-who")
-        return hypervisor_name, guest_name
+    hypervisor_name, guest_name = _get_hypervisor_mapping(logs)
+    # Delete the hypervisor entry and always make sure it's new.
+    hosts = Host.list({'search': hypervisor_name})
+    for index, host in enumerate(hosts):
+        Host.delete({'id': host['id']})
+    runcmd("systemctl restart virt-who")
+    return hypervisor_name, guest_name
 
 
-def deploy_configure_by_command(command, debug=True):
+def deploy_configure_by_command(command, debug=False):
```

Comment:
Any specific reason to move parameter from `debug=True` to `debug=False` ?

Reply:
Most of the time, `debug=True` only used by two cases:
test_positive_deploy_configure_by_id()
test_positive_deploy_configure_by_script()
Because these two cases need to check the rhsm.log message with the DEBUG enable.

Other 6 UI cases don't need to check the DEBUG message in rhsm.log, so I think `debug=False` is a common setting, otherwise, we will have many  statement as `deploy_configure_by_command(command, debug=False)`

Reply:
okay, make sense.